src/styles/components/_accordion-light.scss

```diff
@@ -0,0 +1,76 @@
+$dso-accordion-light-main-color: $grasgroen;
+$dso-accordion-light-chevron-icon-size: $u3;
+$dso-accordion-light-chevron-space: $u1;
+$dso-accordion-light-chevron-left-offset: $dso-accordion-light-chevron-icon-size + $dso-accordion-light-chevron-space;
+$dso-accordion-light-handle-line-height: $u3;
+
+$dso-accordion-light-border-color: $grijs-20;
+
+.dso-accordion-light {
+  border-bottom: 1px solid $dso-accordion-light-border-color;
+
+  .dso-section-handle,
+  > h1.dso-section-handle,
+  > h2.dso-section-handle,
+  > h3.dso-section-handle,
+  > h4.dso-section-handle,
+  > h5.dso-section-handle {
```

Comment:
Volgens mij is alleen regel 12 wel genoeg toch? En dan mag dit riedeltje scss eigenlijk wel naar regel 23 verplaatst worden.

Reply:
aangepast, ook in de normale accordion